packages/shared/src/modules/points-of-interest/libs/enums/points-of-interest-validation-rule.ts

```diff
@@ -1,4 +1,5 @@
 const PointsOfInterestValidationRule = {
+	DESCRIPTION_MAX_LENGTH: 100,
```

Comment:
let's make it larger and also add it to the migration string(1000)
```suggestion
	DESCRIPTION_MAX_LENGTH: 1000,
```